### ISSUE TITLE
CI upload builds shouldn't throw error when no files were created

### ISSUE THREAD

**Issue by jonrkarr** [reproducible-biomedical-modeling/CRBM-Viz#3538]
https://github.com/biosimulations/biosimulations/runs/4141385138?check_suite_focus=true

```yaml
      - name: Upload builds
        uses: actions/upload-artifact@v2
        if: ${{inputs.upload}}
        with:
          name: ${{inputs.name}}
          path: dist/
          retention-days: 5
          if-no-files-found: error
```

**Comment by bilalshaikh42** [comment]
Ah, this was set for cases when we need to deploy the produced outputs. I've made the failure conditional on whether we are building all the apps or just the affected
